mvg/plotting.py

```diff
@@ -210,6 +210,18 @@ def modes_over_time(
     else:
         data["Date"] = pd.to_datetime(data["timestamps"], unit=timeunit)
 
+    # Add an additional row with a future timestamp to ensure the mode
+    # for the last timestamp in the dataset is clearly visible
+    ts_end, ts_prev = data["timestamps"].iloc[[-1, -2]]
+    ts_next = (ts_end - ts_prev) + ts_end
```

Comment:
The distance between two last timestamps is not a good indicator of length of a mode. There can be cases that the distance between last two timestamps are weeks while majority of initial timestamps were just one day apart. This would create a really skewed view of the length of last timestamp. We can either calculate the average distance between consecutive timestamps and use that (which can be very short if there are a lot of timestamps) or use a fixed length in a way that is visually discernible. I suggest we go with the latter approach
```suggestion
    LAST_TS_RATIO = 30
    ts_start, ts_end = data["timestamps"].iloc[[0, -1]]
    ts_next = (ts_end - ts_start)/LAST_TS_RATIO + ts_end
```